app/scripts/controllers/network/provider-api-tests/shared-tests.js

```diff
@@ -447,59 +516,69 @@ export function testsForRpcMethodAssumingNoBlockParam(method) {
  * cacheable.
  *
  * @param method - The name of the RPC method under test.
+ * @param options - the options
+ * @param options.providerType - The type of provider being tested. Either `infura` or `custom`
+ * (default: "infura").
  */
-export function testsForRpcMethodsThatCheckForBlockHashInResponse(method) {
-  it('does not hit Infura more than once for identical requests and it has a valid blockHash', async () => {
+export function testsForRpcMethodsThatCheckForBlockHashInResponse(
+  method,
+  { providerType = 'infura' } = { providerType: 'infura' },
```

Comment:
Nit: we don't need to set the default provider here twice; the second one is redundant.
```suggestion
  { providerType = 'infura' } = {},
```

Reply:
This has been fixed. See: [66aa9c1](https://github.com/MetaMask/metamask-extension/pull/16337/commits/66aa9c1b1c4961ba571948aea065608cb7517087)